virttest/utils_split_daemons.py

```diff
@@ -388,3 +388,21 @@ def virtproxyd_stop():
 def virtproxyd_restart():
     virtproxyd_instance = VirtProxyd()
     return virtproxyd_instance.restart()
+
+
+def is_modular_daemon(session=None):
+    """
+    Check whether modular daemon is enabled
+
+    :params session: An session to guest or remote host
+    :return: True if modular daemon is enabled
+    """
+    daemons = ["virtqemud.socket", "virtinterfaced.socket",
```

Comment:
As I understand, checking virtqemud.socket  status only is enough.

Reply:
Please have a look at 'Libvirt modular deamon follow up', it needs all of them.

Reply:
it is used to detect which mode current libvirtd is working, modular or monothlic ?  not checking all daemons work normally under modular mode.  So only virtqemud.socket check  is enough.
@yafu-1 ,please help check this.

Reply:
Only for modular daemons, there are several ways to check if libvirtd is working, I don't want to add a new way.
Actually, it also needs virtproxyd.socket at least for some cases, so I don't think it's enough to only check virtqemud.socket.

Reply:
@Yingshun @chunfuwen 
It's not robust enough if just checking virtqemud. It can not ensure other split daemons are running besides virtqemud. So it's better to check all the split daemons.

Reply:
Thanks for your updates! @yafu-1

Reply:
@yafu-1 
@Yingshun 
Check all split daemons status doesn't allow cover future possible scenarios where split daemon is started once required.

Reply:
@chunfuwen Hmm, I think that's why checking virtqemud only is not sufficient. Actually we only have a 'full' split daemon scenario for now and decide to restart all sub daemons if need, so I think it's ok. And the purpose of this function is to check if it's a modular daemon env, so I think any() has already fixed your problem. How do you think?